internal/authentication/common.go

```diff
@@ -0,0 +1,115 @@
+package authentication
```

Comment:
nit: Rename to `middleware.go`

Optionally move `GetJWTValidator` to `validator.go` or make it private to this package.

Reply:
Made the change